### PR TITLE
Have the --colored default value be yes in the docs

### LINKED ISSUE #2288: tox4: --colored is yes by default
With tox4a9, contrary to the [docs](https://tox.wiki/en/rewrite/cli_interface.html#tox---colored) `--colored` is yes by default.
What is the expected default?

### PR DESCRIPTION
**Pull request by AntoineD**
Fixes #2288.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -86,6 +87,9 @@
 nitpicky = True
 nitpick_ignore = []
 
+# Force --colored default value to be yes.
+os.environ["FORCE_COLOR"] = "yes"
```
Comment (gaborbernat) [inline docs/conf.py:91]: can you put the command to the right-hand side here?

**Comment by AntoineD** [comment]
Is the changelog fragment really needed here?

**Comment by gaborbernat** [comment]
> Is the changelog fragment really needed here?

We can ignore that now.

File: docs/conf.py (hunk 1 above)
Comment (AntoineD) [reply docs/conf.py:91]: Sorry I do not understand what you mean.

Comment (gaborbernat) [reply docs/conf.py:91]:
```python
os.environ["FORCE_COLOR"] = "yes"   # ensure by default we documented colored
```

Comment (AntoineD) [reply docs/conf.py:91]: I see now (you wrote command instead of comment).

Comment (gaborbernat) [reply docs/conf.py]: my bad :laughing:

File: docs/conf.py [hunk 2]
```diff
@@ -86,6 +87,8 @@
 nitpicky = True
 nitpick_ignore = []
 
+os.environ["FORCE_COLOR"] = "yes"  # Force --colored default value to be yes.
```
Comment (gaborbernat) [inline docs/conf.py:90]: can you make the comment lowercase and remove the period? generally, I don't write it as a sentence unless there's multiple sentences

Comment (AntoineD) [reply docs/conf.py]: Sure (I just tend to follow pep8 by default).

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**